### ISSUE TITLE
metro manila version

### ISSUE THREAD

**Issue by benbancure94** [OSSPhilippines/philippines-travel-level-map#10]
Good day!

I created my own version, but only for Metro Manila. This is not yet pushed into this repo.

![phMap (6)](https://user-images.githubusercontent.com/29687203/232490503-e3140a80-5ea0-453e-903a-78ea75b74c58.jpg)

**Comment by benbancure94** [comment]
I downloaded a clone of this repository and modified in my local computer.

**Comment by warengonzaga** [comment]
Cool, do you have plans to release it to the public or you want to add that as new feature for the project?

**Comment by benbancure94** [comment]
Would you mind that I will release this to public myself but using the cloned code?

**Comment by warengonzaga** [comment]
Hmm, if you want to release it, feel free but please add credits to your readme linking back to this repository.

**Comment by benbancure94** [comment]
Ok, sure .

**Comment by loumexx** [comment]
This is awesome! I'm actually also making my own, pretty much serving as coding practice. Looking forward to when this gets released!

**Comment by benbancure94** [comment]
[Here](https://benbancure94.github.io/philippine-map-app) is mine.

**Comment by jofftiquez** [comment]
> [Here](https://benbancure94.github.io/philippine-map-app) is mine.

Great work! Do you have plans on making a PR for this to the main repo?

**Comment by warengonzaga** [comment]
same question @benbancure94 maybe we can add this to the main project.

**Comment by denzdelvillar** [comment]
I think our options are:

a) we can put a section `Have You Tried:` that lists down links to all other maps like @benbancure94's that was inspired by this project, or
b) integrate the new maps into the main project itself

I'm more inclined towards option `a` because it gives the other developers more freedom with their own maps without having to do a PR on this project when they change something.

**Comment by jofftiquez** [comment]
> I think our options are:
> 
> a) we can put a section `Have You Tried:` that lists down links to all other maps like @benbancure94's that was inspired by this project, or b) integrate the new maps into the main project itself
> 
> I'm more inclined towards option `a` because it gives the other developers more freedom with their own maps without having to do a PR on this project when they change something.

I agree. Our goal to highlight all projects as much as possible while maintaining collaboration at the same time. Kudos to you guys. It's still up to @benbancure94 let us know what you think.

Speaking of option `a`, @denzdelvillar maybe we can start updating the design of the about page? As suggested in https://github.com/OSSPhilippines/philippines-travel-level-map/issues/15 we need to put the map itself first, then additional details in other pages. For the idea of option `a` we can add a toolbar to the `/` page.

**Comment by warengonzaga** [comment]
We can add a menu in the main web app for other versions, what do you folks think?

**Comment by warengonzaga** [comment]
Are we still going to add this? I think we can look back at this again sometime in the future.
